.github/workflows/merge-queue.yml

```
@@ -3,8 +3,100 @@ name: Validate code in the merge queue (post merge)
+    if: needs.prepare.outputs.TEST_CHUNKS
+    runs-on: ec2-runners
+    container:
+      image: public.ecr.aws/m3u4c4h9/island-is/actions-runner-public:latest
+    needs:
+      - prepare
+    env:
+      AFFECTED_PROJECTS: ${{ matrix.projects }}
+      MAX_JOBS: 1
+    strategy:
+      fail-fast: false
+      matrix: ${{ needs.prepare.outputs.TEST_CHUNKS }}
```

actionlint (1.7.4)

type of expression at "matrix" must be object but found type string

(expression)